pkg/apis/core/types.go

```diff
@@ -2741,6 +2741,32 @@ type ContainerStatus struct {
 	// +optional
 	// +featureGate=RecursiveReadOnlyMounts
 	VolumeMounts []VolumeMountStatus
+	// User represents user identitiy information initially attached to the first process of the container
+	// +featureGate=SupplementalGroupsPolicy
+	// +optional
+	User *ContainerUser
+}
+
+// ContainerUser represents user identity information
+type ContainerUser struct {
+	// Linux holds user identity information initially attached to the first process of the containers in Linux.
+	// Note that the actual process identity can be dynamic if the initially attached identity have enough privilege calling setuid/setgid/setgroups syscalls
+	// +optional
+	Linux *LinuxContainerUser
```

Comment:
I emphasize the information is __initially attached__ one here.

As @ktock pointed out in https://github.com/containerd/containerd/pull/9737#discussion_r1598603468, process identity can be dynamic/updated if the initially attached identity has enough privilege calling setuid/setgid/setgroups syscall.